Create ru_ru.json

Heimdallr-1 opened the pull request.

Heimdallr-1 committed:
Create ru_ru.json

wendall911 commented:
Can you get another native speaker to review and confirm the translation. I see a couple things in my limited Russian that don't look correct. For example, why use Кремниевые for flint? Flint is a different type of material than silicon.

Heimdallr-1 committed:
Update ru_ru.json

Fix the confusion between flint and silicon. I'll keep that in mind. Thank you.

Flint (кремень);
Silicon (кремний).
